test/api/liskApi.js

```diff
@@ -130,6 +130,56 @@ describe('Lisk API module', () => {
 				(LSK).should.have.property('testnet').be.equal(true);
 			});
 		});
+
+		describe('on initialize SSL it', () => {
+			it('should set SSL to true', () => {
```

Comment:
These test descriptions don't convey the full context (i.e. that an option is being passed).